Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -458,4 +442,73 @@ class RemoteTabsNotLoggedInCell: UITableViewCell {
+            imgView.snp_remakeConstraints { (make) -> Void in
+                make.centerY.equalTo(contentView.snp_centerY).offset(-65).priorityMedium() // Sets proper top constraint for bigger phones: iPhone 6 and on.
+                make.top.greaterThanOrEqualTo(contentView.snp_top).offset(35).priorityHigh() // Sets proper top constraint for smaller phones: iPhone 4 & 5.
+                make.centerX.equalTo(instructionsLabel)
+            }
+
+            titleLabel.snp_remakeConstraints({ (make) -> Void in
+                make.top.equalTo(imgView.snp_bottom).offset(RemoteTabsPanelUX.EmptyStateTopPaddingInBetweenItems)
+                make.centerX.equalTo(imgView)
+            })
+
+            instructionsLabel.snp_remakeConstraints({ (make) -> Void in
+                make.top.equalTo(titleLabel.snp_bottom).offset(RemoteTabsPanelUX.EmptyStateTopPaddingInBetweenItems)
+                make.width.equalTo(RemoteTabsPanelUX.EmptyStateInstructionsWidth)
+                make.left.lessThanOrEqualTo(contentView.snp_left).offset(80).priorityMedium() // Sets proper landscape layout for bigger phones: iPhone 6 and on.
+                make.right.lessThanOrEqualTo(contentView.snp_centerX).offset(-10).priorityHigh() // Sets proper landscape layout for smaller phones: iPhone 4 & 5.
+            })
+
+            signInButton.snp_remakeConstraints { (make) -> Void in
+                make.height.equalTo(RemoteTabsPanelUX.EmptyStateSignInButtonHeight)
+                make.width.equalTo(RemoteTabsPanelUX.EmptyStateSignInButtonWidth)
+                make.centerY.equalTo(titleLabel.snp_centerY)
+                make.right.greaterThanOrEqualTo(contentView.snp_right).offset(-80).priorityMedium() // Sets proper landscape layout for bigger phones: iPhone 6 and on.
+                make.left.greaterThanOrEqualTo(contentView.snp_centerX).offset(10).priorityHigh() // Sets proper landscape layout for smaller phones: iPhone 4 & 5.
+            }
+
+            createAnAccountButton.snp_remakeConstraints({ (make) -> Void in
+                make.bottom.equalTo(instructionsLabel.snp_bottom)
+                make.centerX.equalTo(signInButton)
+            })
+        } else {
+            imgView.snp_remakeConstraints { (make) -> Void in
+                if DeviceInfo.deviceModel().rangeOfString("iPad") != nil {
+                    make.centerY.lessThanOrEqualTo(contentView).offset(-160) // Sets proper center constraint for iPads.
+                } else {
+                    make.centerY.equalTo(contentView.snp_centerY).offset(-180).priorityMedium() // Sets proper top constraint for iPhone 6 in portait.
+                    make.top.greaterThanOrEqualTo(contentView.snp_top).offset(50).priorityHigh() // Sets proper top constraint for iPhone 4, 5 in portrait.
+                }
+                make.centerX.equalTo(contentView)
+            }
+
+            titleLabel.snp_remakeConstraints({ (make) -> Void in
```

Comment:
If possible, it's best to define constraints just once, rather than destroying and recreating all of them every time `updateConstraints` is called. `titleLabel` is one example where we don't need to change the constraints since the constraint definition is the same in both places.

In general, keep as many constraints constant as possible, moving them out of `updateConstraints`.